com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs

```diff
@@ -176,19 +176,16 @@ static void Drawer_TitleDefaultFrameSettings(SerializedHDRenderPipelineAsset ser
 
         static void Drawer_SectionGeneral(SerializedHDRenderPipelineAsset serialized, Editor owner)
         {
-            EditorGUILayout.PropertyField(serialized.renderPipelineResources, Styles.GeneralSection.renderPipelineResourcesContent);
+            using (new EditorGUI.DisabledScope(true))
+            {
+                EditorGUILayout.PropertyField(serialized.renderPipelineResources, Styles.GeneralSection.renderPipelineResourcesContent);
 
-            HDRenderPipeline hdrp = (RenderPipelineManager.currentPipeline as HDRenderPipeline);
-            if (hdrp != null && hdrp.rayTracingSupported)
-                EditorGUILayout.PropertyField(serialized.renderPipelineRayTracingResources, Styles.GeneralSection.renderPipelineRayTracingResourcesContent);
+                HDRenderPipeline hdrp = (RenderPipelineManager.currentPipeline as HDRenderPipeline);
+                if (hdrp != null && hdrp.rayTracingSupported)
+                    EditorGUILayout.PropertyField(serialized.renderPipelineRayTracingResources, Styles.GeneralSection.renderPipelineRayTracingResourcesContent);
+            }
 
-            // Not serialized as editor only datas... Retrieve them in data
```

Comment:
So I think this means this:
- HDRenderPipelineEditorResources reference is not serialized in the HDRenderPipelineAsset because it is editor only data. We do not want it in build and for this we prefer keep them outside the serialized YAML.
- As it is not serialized, we must seek the targets of the editor to check any different value and such.

Reply:
That's the idea behind this PR. Users wants to customize behavior in editor regarding default material and other such things. To do so we have no choice but to serialize this part too.